### ISSUE TITLE
ExceptionHandlerHelper::render() should Accept Throwable as second argument.

### ISSUE THREAD

**Issue by digitalsounds** [MarcinOrlowski/laravel-api-response-builder#186]
**Please describe the problem**
When overriding the **Illuminate\Foundation\Exceptions\Handler::render()** method in **app\App\Exceptions\Handler**, it accepts **Throwable** as second argument, however when forwarding the second argument onto  **ExceptionHandlerHelper::render()** method, **ExceptionHandlerHelper::render()** second argument only accepts **Exception**.

**Steps to reproduce the behavior:**
1. Update **app\App\Exceptions\Handler::render()** to call **ExceptionHandlerHelper::render()** as per documentation.
2. Create a call to a non existent class in config/response_builder.php, i.e assign a value to e.g MyFakeClass::myFakeMethod();
```
	'converter'         => [
		'primitives' => [
                      MyFakeClass::myFakeMethod(),
...
```
3. Fatal Error is thrown stating the following:
**Fatal error: Uncaught TypeError: MarcinOrlowski\ResponseBuilder\ExceptionHandlerHelper::render(): Argument #2 ($ex) must be of type Exception, Error given, called in /app/app/Exceptions/Handler.php on line 55 and defined in /app/vendor/marcin-orlowski/laravel-api-response-builder/src/ExceptionHandlerHelper.php on line 40**

**Expected behavior**
Update **ExceptionHandlerHelper::render()** to accept **Throwable** as second argument instead of **Exception**.

**Runtime environment**
 - Ubuntu 20.04
 - PHP  8.0
 - Laravel 8.12

**Notes**
Previously Illuminate\Foundation\Exceptions\Handler::render() method DID accept Exception, but was changed in Laravel 7.x
https://laravel.com/docs/7.x/upgrade#symfony-5-related-upgrades.

**Comment by MarcinOrlowski** [comment]
Thanks for reporting the issue. I created PR #187 that shall address this. Please try it out (it's relatively small) - just grab `ExceptionHandlerHelper.php` and replace currently used one with it.

**Comment by MarcinOrlowski** [comment]
Fixed in v9.2.1
